health-monitors/gpu-health-monitor/gpu_health_monitor/tests/test_platform_connector/test_platform_connector.py

```diff
@@ -19,11 +19,19 @@
 import grpc
 import time
 import unittest
+import sys
+import os
 from typing import Any
 from concurrent import futures
 from gpu_health_monitor.dcgm_watcher import types as dcgmtypes
 from gpu_health_monitor.platform_connector import platform_connector
-from gpu_health_monitor.platform_connector.protos import platformconnector_pb2, platformconnector_pb2_grpc
+
+# Add data-models to Python path for protobuf imports
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", "..", "data-models"))
```

Comment:
Since we have only 1 python module and we don't want to add more, can we just keep the models in the python project?